### PR TITLE
fix: navigation tree working

### PR DESCRIPTION
**Pull request by DmaKill**
Resolves #3559
Impact: **major**
Type: **bugfix**

## Issue

The Navigation page displays a blank area instead of the navigation tree, making it impossible to see or edit the navigation tree.

## Solution

Update packages
"react-dnd-html5-backend": "14.0.0"
"react-dnd": "14.0.0"
"react-sortable-tree": "^ 2.8.0"

The parent DndProvider is assigned in options the rootElement (body) parameter, so that its children can self-assign the provider instance created, in each case that drag-and-drop is used, it must be enclosed in the DndProvider tag,

Navigation

### CODE REVIEW

**Comment by DmaKill** [comment]
@Akarshit @MohanNarayana

**Comment by jhonsfran** [comment]
@loan-laux

**Comment by MohanNarayana** [comment]
@DmaKill Thanks for submitting the PR:smiley:. I saw your message regarding the checks failing on the issue. I will require the DCO signoff from your end to legally accept your code. Please look into the 'Details' link for the DCO check fail to find steps to amend the commit. Also, if you can pull the latest branch to resolve any merge conflicts, that should fix the build too. Let me know if you need anything. Thanks

File: imports/plugins/core/navigation/client/components/NavigationTreeContainer.js
Comment (MohanNarayana) [inline imports/plugins/core/navigation/client/components/NavigationTreeContainer.js]: Please remove commented line.

Comment (DmaKill) [reply imports/plugins/core/navigation/client/components/NavigationTreeContainer.js]: Ok, I'm going to extract the last branch and remove the comment

**Comment by MohanNarayana** [comment]
@DmaKill I tested the fix and it works great. Its good to merge once the checks pass. Thanks for working on this.

**Comment by DmaKill** [comment]
@MohanNarayana With [@andresmv](https://github.com/andresmv94) we have solved the conflicts, but we have not been able to with DOC, and test unit

**Comment by Akarshit** [comment]
Once those are taken care of, we would be happy to merge this.

**Comment by MohanNarayana** [comment]
@DmaKill are [these](https://github.com/reactioncommerce/reaction-admin/pull/382/checks?check_run_id=3165372630) steps helpful? Ensure you have your `user.name` and `user.email` in git config.

**Comment by loan-laux** [comment]
Thanks a lot @DmaKill! Couldn't respond or review your PR because I was on holiday. Nice job! I'm looking forward to it getting merged.

**Comment by loan-laux** [comment]
@DmaKill I believe your commits `fe80627` and `29b4b95` aren't signed off, which is why DCO is failing. About the failing unit tests, does anyone from the core team have any idea? @MohanNarayana @Akarshit

**Comment by Akarshit** [comment]
Hmm, couldn't see easily why the tests are failing. Will look into in on Monday.

**Comment by Akarshit** [comment]
@DmaKill If you update "@wojtekmaj/enzyme-adapter-react-17" to "^0.6.3" and "react-dom" to "^17.0.2" the test case should start passing.

**Comment by Akarshit** [comment]
There is something wrong with your package-lock.json file. I see the following error on CircleCI:
`Invalid: lock file's @wojtekmaj/enzyme-adapter-react-17@0.6.2 does not satisfy @wojtekmaj/enzyme-adapter-react-17@^0.6.3`

`Invalid: lock file's react-dom@16.14.0 does not satisfy react-dom@17.0.2`

**Comment by DmaKill** [comment]
@MohanNarayana yeah, user.name and user.email in git config
Captura de pantalla de 2021-07-29 11-19-52

**Comment by loan-laux** [comment]
@DmaKill for DCO, I believe your newer commits aren't signed off anymore. Check `7a6c022` for example.

**Comment by DmaKill** [comment]
@loan-laux the commit I do it this way `git commit -s -m "fix: dco"`, but it doesn't work

**Comment by DmaKill** [comment]
@loan-laux check this commit [87715b1](https://github.com/reactioncommerce/reaction-admin/pull/382/commits/87715b10c56e4d38ef5a528a7b0656c6506ac972)

**Comment by loan-laux** [comment]
@DmaKill there's still a few commits that aren't signed off:

- `fe80627`
- `29b4b95`
- `7a6c022`
- `3989953`

Thanks for your patience with this. And for future contributions, set your `git` to auto-sign commits so that you don't have to worry about it anymore!

**Comment by DmaKill** [comment]
@Akarshit done test unit, DCO is missing

**Comment by Akarshit** [comment]
Maybe these instructions can help: https://github.com/reactioncommerce/reaction-admin/pull/382/checks?check_run_id=3223154554

**Comment by Akarshit** [comment]
@DmaKill I squashed your commits into one(that was signed) and pushed to your branch. I hope that's okay.

**Comment by Akarshit** [comment]
@MohanNarayana Can you check this one again please. If nothing got broken, we can get this merged.

**Review by MohanNarayana — APPROVED** [review]
Great job. Thanks for your contribution.

**Pull request merged**
